Web/Assignment-2/EmployaaDetails/src/EmployeeData.java

```diff
@@ -0,0 +1,65 @@
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class EmployeeData
+ */
+@WebServlet("/EmployeeData")
+public class EmployeeData extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * @see HttpServlet#HttpServlet()
+	 */
+	public EmployeeData() {
+		super();
+		// TODO Auto-generated constructor stub
+	}
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse
+	 *      response)
+	 */
+	protected void doGet(HttpServletRequest request,
+			HttpServletResponse response) throws ServletException, IOException {
+		// TODO Auto-generated method stub
+		String empName = request.getParameter("name");
+
+		String detail = "";
+		switch (empName) {
+		case "Anshita":
+			detail = "Name: Anshita Arya Age:22 Dob:19-09-1992  Address:Bihar";
```

Comment:
ACTION: You have created a Employee class to maintain information but didn't use it. In this web method you should return Employee object.